packages/serverpod_lints/lib/internal.yaml

```diff
@@ -18,9 +18,3 @@ linter:
     sort_pub_dependencies: true
     unawaited_futures: true
     unnecessary_final: false
-
-analyzer:
-  language:
-    strict-casts: true
-    strict-inference: true
-    strict-raw-types: true
```

Comment:
It is odd that we enable it in the serverpod_lints package, only to disable it everywhere we use it (except relic). 

I do think it was/is supposed to be a temporary measure, and that _we_ intended/intends for all code to align with the `serverpod_lints` package eventually.

However, I also think we have become wiser since. Perhaps we should revisit all the extra rules enabled in `serverpod_lints` and decide if they carry their weight.

I approve this PR, but I suggest we discuss the lints in general in a wider forum.